src/math/Box2.d.ts

```diff
@@ -16,16 +16,16 @@ export class Box2 {
 	copy( box: Box2 ): this;
 	makeEmpty(): Box2;
 	isEmpty(): boolean;
-	getCenter( target: Vector2 ): Vector2;
-	getSize( target: Vector2 ): Vector2;
+	getCenter( target?: Vector2 ): Vector2;
+	getSize( target?: Vector2 ): Vector2;
```

Comment:
I'm afraid these changes are wrong. `target` is mandatory. Checks like:
```js
if ( target === undefined ) {

	console.warn( 'THREE.Box3: .getSize() target is now required' );
	target = new Vector3();

}
```
are for backwards compatibility only.

Reply:
En... May be this `Vector3.d.ts cross()` method should be corrected

https://github.com/mrdoob/three.js/blob/b4473c25816df4a09405c7d887d5c418ef47ee76/src/math/Vector3.d.ts#L214

https://github.com/mrdoob/three.js/blob/b4473c25816df4a09405c7d887d5c418ef47ee76/src/math/Vector3.js#L513-L524

Reply:
Yes, `w?: Vector3` should be removed.

Reply:
> Yes, `w?: Vector3` should be removed.

OK.